### PR TITLE
Job feed

### PR DESCRIPTION
**Pull request by atanasovRazvan**
Completed Job Feed with tests and styles

### CODE REVIEW

File: src/job-feed/Job.js [hunk 1]
```diff
@@ -0,0 +1,77 @@
+import React from 'react';
+import {
+  Button, Card, CardActions, CardContent, Typography,
+} from '@material-ui/core';
+import PropTypes from 'prop-types';
+import './styles.css';
+import { KeyboardArrowRight } from '@material-ui/icons';
+
+const Job = ({
```
Comment (ih90) [inline src/job-feed/Job.js]: I would call this JobCard

File: src/job-feed/JobList.js [hunk 2]
```diff
@@ -0,0 +1,34 @@
+import React, { useEffect, useState } from 'react';
+import { useQuery } from '@apollo/client';
+import { GET_JOBS } from '../sources';
+import Job from './Job';
+import './styles.css';
+
+const JobList = () => {
+  const [jobList, setJobList] = useState([]);
```
Comment (ih90) [inline src/job-feed/JobList.js]: Why do you need to have the jobs in the state?

File: src/job-feed/JobList.js [hunk 3]
```diff
@@ -0,0 +1,34 @@
...
+      setJobList(data.jobs);
+    }
+  }, [data]);
+
+  return (
+    <div className="jobs-container">
+      {error ? null : (jobList && jobList.map((job) => (
+        <Job
+          key={job.id}
+          companyName={job.company.name}
```
Comment (ih90) [inline src/job-feed/JobList.js]: When reading values deep in the chain is a good practice to check first if they exists. For example if the company name does not exists, your whole component will crash. See https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Operators/Optional_chaining

File: src/job-feed/__tests__/JobList.test.js [hunk 4]
```diff
@@ -0,0 +1,54 @@
...
+describe('tests for JobList', async () => {
+  beforeEach(async () => {
+    render(
+      <MockedProvider mocks={mocks}>
+        <JobList />
+      </MockedProvider>,
+    );
+  });
+
+  it('should see job details on the page', async () => {
```
Comment (ih90) [inline src/job-feed/__tests__/JobList.test.js]: I don't see much difference between this tests and the one for the job card. Here I would test that all the N jobs are displayed. We should care here about the list, not what an entry in that list might contain (already tested in another suite).

File: src/job-feed/Job.js (hunk 1 above)
Comment (atanasovRazvan) [reply src/job-feed/Job.js]: updated the name

File: src/job-feed/JobList.js (hunk 2 above)
Comment (atanasovRazvan) [reply src/job-feed/JobList.js]: overthinking
fixed

File: src/job-feed/JobList.js (hunk 3 above)
Comment (atanasovRazvan) [reply src/job-feed/JobList.js]: fixed and improved for jobList && jobList.map to jobList?.map

File: src/job-feed/__tests__/JobList.test.js (hunk 4 above)
Comment (atanasovRazvan) [reply src/job-feed/__tests__/JobList.test.js]: updated

**Review by ih90 — APPROVED** [review]

**Pull request merged**